util/localconfig/localconfig.go

```diff
@@ -79,7 +79,17 @@ func (u *User) Claims() (*jwt.RegisteredClaims, error) {
 func ReadLocalConfig(path string) (*LocalConfig, error) {
 	var err error
 	var config LocalConfig
+
+	if fi, err := os.Stat(path); err == nil {
```

Comment:
Should we also return an error if we can't stat the file?

Reply:
No, we can't return err, as in some cases, the config file hasn't been created yet on the disk. I'm checking the file permission only when the file exist on the disk